js/ui/scheduler/rendering_strategies/ui.scheduler.appointments.strategy.base.js

```diff
@@ -186,20 +186,11 @@ class BaseRenderingStrategy {
     }
 
     _getAppointmentCoordinates(itemData) {
-        let coordinates = [{
```

Comment:
Does this old logic contained option to fall back to zero coordinates if `needCoordinates` will fail with exception?